android/keyple-plugin/android-nfc/src/test/kotlin/org/eclipse/keyple/plugin/android/nfc/AndroidNfcReaderImplTest.kt

```diff
@@ -24,13 +24,15 @@ import io.mockk.unmockkAll
 import java.io.IOException
 import org.eclipse.keyple.core.service.exception.KeypleReaderException
 import org.eclipse.keyple.core.service.exception.KeypleReaderIOException
+import org.eclipse.keyple.plugin.android.nfc.smartremoval.AndroidNfcReaderImpl
 import org.junit.After
 import org.junit.Assert
 import org.junit.Before
 import org.junit.FixMethodOrder
 import org.junit.Test
 import org.junit.runner.RunWith
 import org.junit.runners.MethodSorters
+import org.robolectric.Robolectric
```

Comment:
This org.robolectric third dependency should be declared to Eclipse
https://dev.eclipse.org/ipzilla/buglist.cgi?component=iot.keyple